ci/ubuntu.dockerfile

```diff
@@ -7,20 +7,37 @@ ARG venv=venv
 
 RUN echo "To invalidate cache"
 
+
+ RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys CE7709D068DB5E88
+ RUN add-apt-repository "deb https://repo.sovrin.org/ursa/deb xenial master"
+
 RUN apt-get update -y && apt-get install -y \
     python3-nacl \
-    libindy-crypto=0.4.5 \
+
+    cmake \
```

Comment:
Can you please provide more info why do we need this new dependencies explicitly? If Ursa depends on something, should it automatically install dependencies?

Reply:
It seems to me that most of this is required for the package compilation that is happening below, and is not tied to Ursa directly. If we remove the compilation step and use the libsodium that is already installed on the system I think most if not all of these could be removed.

```
└─ $ ▶ apt show python3-nacl | grep -i depends
Depends: python3-cffi, python3 (<< 3.6), python3 (>= 3.5~), python3-cffi-backend-api-9729, python3-six, libc6 (>= 2.4), libsodium18 (>= 0.7.0)
```

Others should be put into debian package dependencies.

Reply:
Correct. Ursa does not depend on libsodium. Just OpenSSL.